src/main/kotlin/org/rust/cargo/project/configurable/CargoConfigurable.kt

```diff
@@ -26,6 +26,12 @@ class CargoConfigurable(project: Project) : RsConfigurableBase(project, "Cargo")
             CargoCommandCompletionProvider(project.cargoProjects, "check ") { null }
         )
         return panel {
+            row {
+                checkBox(
+                    "Automatically show first error in editor",
```

Comment:
This option name looks ok in Java compiler settings because Java compiler just compiles a code, so it's obvious that 
"Automatically show first error in editor" means "if compilation fails". But Cargo does a lot of things, so this is a very confusing name in our case.  Let's change it to something like "Automatically show first error in editor after a build failure"